src/components/social-follow-links/social-follow-links.spec.js

```diff
@@ -30,7 +30,7 @@ describe('Social Follow Links component', () => {
   it('should have SlackIcon link', () => {
     const slackLink = socialFollowLinks.find('.social-container .slack-link');
 
-    expect(slackLink.prop('href')).toEqual('http://providencegeeks.slack.com/');
+    expect(slackLink.prop('href')).toContain('https://join.slack.com/t/providencegeeks/shared_invite/');
```

Comment:
thought using `toContain` might make more sense here, since the link itself is so long and cumbersome.